### ISSUE TITLE
FE8: Final Chapter Reinforcement Issue

### ISSUE THREAD

**Issue by orangejuice8** [lushen124/Universal-FE-Randomizer#179]
First off: this was in Hard Mode, Ephraim's Route. I played using the OpenEmu GBA emulator. Randomizer settings I used that might be relevant are:
Randomize Minions - Yes
Mix Monster and Human Classes - No
On roughly Turn 14 of _Final Chapter: Sacred Stone_, three enemy reinforcement units appeared on the stairs at the uppermost edge of the map. One was a regular Elder Bael. However, there was also a monster unit with 0 in all stats and a blank class name with the ballista icon, and another monster with the soldier icon, 0 in all stats (except for a 4 in skill) and Eirika's portrait. Interestingly, the latter created a Magic Seal effect in a 10-tile radius around it (meaning neither tomes nor staves can be used in this area). The effect faded when this enemy was killed. Pictures have been attached.
<img width="1599" alt="Error 1" src="https://user-images.githubusercontent.com/51896129/59575317-315a7d80-9078-11e9-9198-fa5a20476cb6.png">
<img width="1599" alt="Error 2" src="https://user-images.githubusercontent.com/51896129/59575324-391a2200-9078-11e9-8672-2554666c7057.png">
<img width="1599" alt="Error 3" src="https://user-images.githubusercontent.com/51896129/59575331-3dded600-9078-11e9-85e5-d6c8c708b4d1.png">

**Comment by lushen124** [comment]
Weird. If you have a changelog, you can see what the monsters became in the Chapter section of the changelog. It's a bit hard to parse, but it lists all units the randomizer could have modified for each chapter.

**Comment by lushen124** [comment]
Closing until we have more details or a changelog to debug.
